tests/test_exprs_as_stmts.py

```diff
@@ -0,0 +1,40 @@
+import sys
+import unittest
+
+from better_exchook import better_exchook
+
+from tests.run_examples_dir import make_example_cases
+
+
+def test_examples():
+  for x in make_example_cases('examples_exprs_as_stmts'):
+    yield x
+
+
+if __name__ == "__main__":
```

Comment:
Add every new test file [to our Github CI config](https://github.com/Zettelkasten/sleepy/blob/main/.github/workflows/main.yml#L24)

Reply:
Or maybe just make this a second function in `test_examples.py` instead